packages/mergebot/src/basic.ts

```diff
@@ -6,8 +6,22 @@ export type ColumnName =
   | "Needs Author Action"
   | "Recently Merged"
   | "Waiting for Code Reviews"
+  | BlessedColumnName
   | "*REMOVE*"; // special value: indicates closing the PR
 
+export type BlessedColumnName = (typeof blessedColumnNames)[number];
+const blessedColumnNames = [
+  "Waiting for Code Reviews (Blessed)",
+] as const;
+
+export function isBlessedColumnName(name: string): name is BlessedColumnName {
```

Comment:
are we ever going back to multiple blessed columns? This seems like overkill otherwise. for example, "Waiting for Code Review (Blessed)" is hard-coded all over compute-pr-actions.ts but not pr-info.ts. Why not both?

Reply:
Probably not, but an earlier revision of the PR did. I can simplify this.